index.js

```diff
@@ -48,9 +48,13 @@ Guard.prototype = {
       }
 
       if (!Array.isArray(permissions)) {
-        return next(new UnauthorizedError('permissions_invalid', {
-          message: 'Permissions should be an Array. Bad format?'
-        }))
+        if (typeof permissions === 'string') {
```

Comment:
I think makes more sense to check if the input value is a string before we check if the input is an Array so we can parse it and always work with an Array value

```javascript
if (typeof permissions === 'string') {
  permissions = permissions.split(' ')
}
// continue with Array
```